libs/dal/src/repositories/job/job.entity.ts

```diff
@@ -27,5 +27,7 @@ export class JobEntity {
   _templateId: string;
   digest?: {
     events?: any[];
+    amount?: number;
+    unit?: string;
```

Comment:
Could you elaborate on adding this two here? As opposed to fetching them from the step entity in runtime?

Reply:
`As opposed to fetching them from the step entity in runtime?`
Yes exactly 👍